include/micm/solver/rosenbrock.hpp

```diff
@@ -153,8 +154,8 @@ namespace micm
       double final_time_{};
     };
 
-    const System system_;
-    const std::vector<Process> processes_;
+    System system_;
+    std::vector<Process> processes_;
```

Comment:
can these no longer be `const`?

Reply:
No, because you can't set base class member variables with this type of syntax:

```
class Derived : Base {
    Derived(int a, int b): a_(a), b_(b){}
};
```

At least, I wasn't able to. I think normally you would just call the base class constructor, but in this case I didn't want the behavior of the base class constructor (setting the jacobian and stuff based off of the processes, which were redundant and not needed here. It would be better to figure out how to make that base solver class like you mentioned in #211.

Reply:
ah, ok - that makes sense